src/middleware/middleware.ts

```diff
@@ -46,5 +48,12 @@ const middleware: Middleware = async ({ app, query, route, $pinia }) => {
   const featureFlagStore = useFeatureFlagStore($pinia)
   featureFlagStore.initFromCookies(app.$cookies.get('features') ?? {})
   featureFlagStore.initFromQuery(query)
+
+  /* UI store */
+
+  const uiStore = useUiStore($pinia)
+  const isMobileUa = app.$ua ? app.$ua.isMobile : false
```

Comment:
It is expected that `isMobileUa` can be set once per request, and we do not need to change its value or update a cookie after that.